plugins/Marketplace/Plugins.php

```diff
@@ -430,14 +430,14 @@ private function addPluginCoverImage(&$plugin): void
      * @param $plugin
      * @return void
      */
-    private function prettifyNumberOfDownloads(&$plugin): void
+    protected function prettifyNumberOfDownloads(&$plugin): void
```

Comment:
Is this visibility change needed? I see in the test you set the accessibility to true anyway.

Reply:
Oh yeah, good one @michalkleiner. Reverted it back to private, thanks!